lib/rototiller/task/rototiller_task.rb

```diff
@@ -77,13 +78,13 @@ def add_flag(*args, &block)
       # for block {|a| ... }
       # @yield [a] Optional block syntax allows you to specify information about command, available methods track hash keys
       def add_command(args={}, &block)
-        attributes = [:name, :override_env]
+        attributes = [:name, :override_env, :argument, :argument_override_env]
```

Comment:
hmmmm.  should this not have its own method to match add_command, add_env, add_flag?
as a user i'd expect to be able to use add_arg, just like flags.